src/parsing/Parse_rule.ml

```diff
@@ -483,6 +480,10 @@ let parse_paths env key value =
     ( take_opt paths_dict env parse_string_list "include",
       take_opt paths_dict env parse_string_list "exclude" )
   in
+  if Hashtbl.length paths_dict.h > 0 then
```

Comment:
maybe add a comment like (* alt: we could use report_unparsed_fields(), but better to raise an error for now
to be compatible with pysemgrep *)